openpathsampling/snapshot.py

```diff
@@ -117,306 +91,73 @@ def copy(self):
         content (e.g. Configuration object) will not.
 
         """
-        this = AbstractSnapshot(is_reversed=self.is_reversed, topology=self.topology)
+        this = self.__class__.__new__(self.__class__)
+        AbstractSnapshot.__init__(this, topology=self.topology)
         return this
 
-    def reversed_copy(self):
-        """
-        Returns a shallow reversed copy of the instance itself. The
-        contained configuration and momenta are not copied and the momenta
-        are marked reversed.
-
-        Since Snapshots are made in pairs this will also lead to a new
-        (non-)reversed copy!
-
-        Returns
-        -------
-        Snapshot()
-            the reversed shallow copy
-        """
+    def create_reversed(self):
+        this = self.copy()
+        this._reversed = self
+        return this
 
-        obj = self.copy()
-        return obj.reversed
 
+class FeatureSnapshot(AbstractSnapshot):
```

Comment:
I thought about just having one class. The reason is that I wanted to still allow to have snapshots that do not use features (as before) the common basis is `AbstractSnapshot`. And FeatureSnapshot is meant to identify Snapshots with features. 

I guess we could get rid of it, but this way the separation is clearer.